index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="stylesheets/main.css" />
+
+    <title>2005 Toyota Corolla S Start Up/ Brief Walkthrough - YouTube</title>
+  </head>
+
+  <body>
+    <header id="main-header">
+      <div id="header-left-section">
+        <i class="fas fa-bars"></i>
+        <img id="logo" src="./images/logo.png" alt="Youtube Logo" />
+      </div>
+      <div id="header-middle-section">
+        <form action="#" id="search-form">
+          <input type="text" placeholder="Search" />
+          <button><i class="fas fa-search"></i></button>
+        </form>
+      </div>
+
+      <div id="header-right-section">
+        <i class="fas fa-video"></i>
+        <i class="fas fa-th"></i>
+        <i class="fas fa-bell"></i>
+        <img
+          src="https://yt3.ggpht.com/a/AATXAJxi-vx7V6HLPGGsFWw_PGiuSs_IRi1B-YAj2A=s176-c-k-c0xffffffff-no-rj-mo"
+          alt=""
+        />
+      </div>
+    </header>
+    <main class="container" id="videos">
+      <section id="left-section">
+        <section id="main-video">
+          <section id="video">
+            <iframe
+              width="560"
+              height="315"
+              src="https://www.youtube.com/embed/BDtGXMhiooM"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </section>
+
+          <section id="video-footer">
+            <section id="videos-footer-upper">
```

Comment:
[REQUIRED !!] HTML file is not validated properly. There are some warnings. Please remove them and resubmit again for another review. [pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png
![Screenshot from 2020-03-27 00-15-58](https://user-images.githubusercontent.com/37747619/77684938-c3f25a80-6fc2-11ea-8305-2e2dc4be7f06.png)
)
TIPS: Use div **tag** instead of **section**.
You guys are a step closer to completion of the first project. Best of Luck.

Reply:
All corrections have been made and the code has passed all validator tests